### PR TITLE
Configure model at runtime

### PR DESCRIPTION
**Pull request by phmarti**
Replace `#if 0` on output files with runtime configuration via parameters.cfg

### CODE REVIEW

File: src/Model/Boussinesq/Sphere/RTC/IRTCModel.cpp [hunk 1]
```diff
@@ -230,79 +230,65 @@ namespace RTC {
       spVis->addHdf5OutputFile(spOut);
    }
 
+   std::map<std::string, std::map<std::string,int> > IRTCModel::configTags() const
+   {
+      std::map<std::string,int> onOff;
```
Comment (castigli) [inline src/Model/Boussinesq/Sphere/RTC/IRTCModel.cpp:235]: maybe I'm missing something, but why maps and not simple enums?

**Comment by phmarti** [comment]
Which part did you want an enum?

File: src/Model/Boussinesq/Sphere/RTC/IRTCModel.cpp (hunk 1 above)
Comment (castigli) [reply src/Model/Boussinesq/Sphere/RTC/IRTCModel.cpp:235]: I might be not understanding the logic, but I meant something like
```c++
enum options { enable, numbered, only_every };
```

Comment (phmarti) [reply src/Model/Boussinesq/Sphere/RTC/IRTCModel.cpp:235]: Enums are always a problem when you need to add something. I didn't want to add one more list of things that need to be in sync, but are "free floating".  And that would only remove the std::string, we still need a map if  you want to set the default. Unless we force the default to be the same for everything.

Comment (phmarti) [reply src/Model/Boussinesq/Sphere/RTC/IRTCModel.cpp:235]: I don't really have a strong opinion for or against it. The general setup (runtime config for IO) is needed to simplify/ guarantee the correct output for the benchmar runs

**Review by castigli — APPROVED** [review]

**Pull request merged**